MANIFEST.in

```diff
@@ -1,20 +1,23 @@
-include src/gluonts/ext/r_forecast/R/*.R
-recursive-include requirements *.txt
+
+# include all of gluonts
+graft src/gluonts
+
+# except the nursery
+prune src/gluonts/nursery
+
+graft docs
+graft requirements
+graft test
+
+includes *.md
+include NOTICE
```

Comment:
does LICENSE not need to be here too?

Reply:
It's picking it up from `setup.cfg`: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#metadata